app/assets/stylesheets/single-post-view.scss

```diff
@@ -142,7 +142,7 @@
 
   .comments > .comment,
   .comment.new-comment-form-wrapper {
-    padding: 10px;
+    padding: 10px 10px 0 10px;
```

Comment:
- Shorthands of length `4` are not allowed. Value was `10px 10px 0 10px`
- Shorthand form for property `padding` should be written more concisely as `10px 10px 0` instead of `10px 10px 0 10px`